### ISSUE TITLE
About DA challenge for generic commitment type

### ISSUE THREAD

**Issue by zhiqiangxu** [ethereum-optimism/developers#519]
I'm interested in understanding more about the current state of DA challenges. 

From what I've gathered, it appears that DA challenges are currently feasible for keccak256 commitments, but it seems they're impossible for generic commitments. 

Is this understanding is correct? If it's true, does it mean that most alt-DAs like celestia will not be DA challenge-able?

**Comment by tynes** [comment]
This is correct @zhiqiangxu. Celestia and EiganDA will have their own security mechanisms

**Comment by zhiqiangxu** [comment]
@tynes So using Celestia and EiganDA for plasma implies some trusting assumptions about the DA, and since DA challenge is not designed for them, in the worst case the data withholding issue may happen, is this correct?
